webpack/common/Switcher/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Switch, Level, LevelItem } from '@patternfly/react-core';
+import {
+  OverlayTrigger,
+  Tooltip,
+  Icon,
+  Grid,
+  ControlLabel,
+  noop,
+} from 'patternfly-react';
```

Comment:
Can you use pf4?

Reply:
You are absolutely right about that, we definitely should!
but when I change it to PF4, it's a bit inconsistent with the [Katello modal](https://github.com/Katello/katello/blob/master/webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js#L3) that uses React-bootstrap which is PF3 base too.

adding here @johnpmitsch, I can do a follow-up PR also to Katello to change those imports to PF4,
and then update our design to match it. how's that sound @mariaaga?

Reply:
Does using all the pf4 components make it look inconsistent with Katello or just `ControlLabel`? Anyway a follow-up PR sounds good :)

Reply:
Yeah, not the same grid system, icon etc..

Reply:
Sorry for the delay! :turkey: 

I've been using PF3 with existing PF3 components, I also found PF4 components to look very out of place on a PF3 page.

IMO any update to PF4 would have to be done to the whole modal, or even the whole subscription page. If you want to update everything go for it, but I think things can stay PF3 or react-bootstrap for now as long as it looks consistent.